prime-router/src/main/kotlin/common/Environment.kt

```diff
@@ -26,6 +26,7 @@ enum class Environment(
     ),
     TEST("test", URL("https://test.prime.cdc.gov"), oktaApp = OktaCommand.OktaApp.DH_TEST),
     STAGING("staging", URL("https://staging.prime.cdc.gov"), oktaApp = OktaCommand.OktaApp.DH_STAGE),
+    DEMO1("demo1", URL("https://demo1.prime.cdc.gov"), oktaApp = OktaCommand.OktaApp.DH_STAGE),
```

Comment:
I know we've currently only got preview and prod for Okta apps, but do we want this to be DH_STAGE and not DH_TEST or DH_DEV?

Reply:
The environment is already configured to connect to staging.

Reply:
Yes, I know. DH_TEST and DH_DEV are both also currently connected to oktaPreview. What I was saying was, should that change, would we expect DEMO1 to continue to connect to oktaPreview, or would we be expecting it to connect to a TEST or DEV instance of Okta?

Reply:
So I think stage makes sense, since we want to keep this close to PROD as possible.  I figured devops had made that decision originally too...

Reply:
Also we do not have a TEST or DEV instance of Okta -- just staging and prod. Not saying we couldn't or shouldn't, but that would be a lot to manage I think for no clear benefit at this time.